tzrec/models/model.py

```diff
@@ -236,3 +236,24 @@ def forward(
         """
         batch = self.get_batch(data, device)
         return self.model.predict(batch)
+
+
+class ScriptWrapperAOT(ScriptWrapper):
+    """Model inference wrapper for aot export."""
+
+    # pyre-ignore [14]
+    def forward(
+        self,
+        data: Dict[str, torch.Tensor],
```

Comment:
the aot model predict may not support device, it need to workaround in predict such as https://github.com/alibaba/TorchEasyRec/blob/master/tzrec/main.py#L1076

Reply:
it is not fully done, currently just commit the export part, the compile part and prediction part are waited to be done